### PR TITLE
create modal component, styles

### PR DESCRIPTION
**Pull request by mialav24**
modal-component

### CODE REVIEW

File: src/Components/App/App.tsx [hunk 1]
```diff
@@ -46,12 +56,17 @@ const App = () => {
           placeholder="Type your category..."
         />
         {
-          <Button disabled={categoryTitle.length >= 3 ? false : true}>
+          <Button
+            type="submit"
+            disabled={categoryTitle.length >= 3 ? false : true}
```
Comment (jlmonroy13) [inline src/Components/App/App.tsx:61]: would be better to write this in this way

`disabled={!(categoryTitle.length >= 3)}`

File: src/Components/Modal/Modal.styles.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+`;
+
+export const ModalContainer = styled.div`
+  background: #f2f2f2;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  height: auto;
+  min-height: 0;
```
Comment (jlmonroy13) [inline src/Components/Modal/Modal.styles.ts:23]: Do we need this?

File: src/Components/Modal/Modal.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  ButtonContainer,
+  FormContainer,
+  ModalContainer,
+  ModalOverlay,
+  TextArea,
+} from "./Modal.styles";
+
+type ModalProps = {
+  closeModal?: () => void;
+  categories: { id: string; title: string }[] | [];
```
Comment (jlmonroy13) [inline src/Components/Modal/Modal.tsx:13]: Create a type called `Category` and use it here

`cateagories: Category[] | []`

File: src/Components/Modal/Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+};
+
+const Modal = ({ closeModal, categories }: ModalProps) => {
+  return (
+    <ModalOverlay>
+      <ModalContainer>
+        <Input placeholder="Type your task's title..." />
+        <TextArea placeholder="Type your task..." name="" id=""></TextArea>
```
Comment (jlmonroy13) [inline src/Components/Modal/Modal.tsx:21]: TextArea doesn't need a close tag

File: src/Components/Modal/Modal.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        <FormContainer>
+          <select id="">
```
Comment (jlmonroy13) [inline src/Components/Modal/Modal.tsx:23]: what is this id for?

**Review by jlmonroy13 — CHANGES_REQUESTED** [review]

File: src/Components/App/App.tsx (hunk 1 above)
Comment (mialav24) [reply src/Components/App/App.tsx:61]: ok, SAPO HPTA!.

File: src/Components/Modal/Modal.styles.ts (hunk 2 above)
Comment (mialav24) [reply src/Components/Modal/Modal.styles.ts:23]: you're right, deleted lines 22 and 23, just needed 20 and 21.

File: src/Components/Modal/Modal.tsx (hunk 4 above)
Comment (mialav24) [reply src/Components/Modal/Modal.tsx:21]: ok

**Review by jlmonroy13 — APPROVED** [review]

**Pull request merged**